### PR TITLE
Restructure social media links for at

### PR DESCRIPTION
**Pull request by RoyEJohnson**
For: https://github.com/openstax/unified/issues/2601
The last commit is just Prettier reformatting. The actual changes are all in the first commit.

### CODE REVIEW

File: src/app/components/Footer/styled.tsx [hunk 1]
```diff
@@ -86,7 +92,9 @@ export const boxed = css`
 `;
 
 const contentWrapperAndNavWidthBreakpoint =
-  '(max-width: ' + remsToEms(contentWrapperMaxWidth + verticalNavbarMaxWidth * 2) + 'em)';
+  '(max-width: ' +
+  remsToEms(contentWrapperMaxWidth + verticalNavbarMaxWidth * 2) +
+  'em)';
```
Comment (jivey) [inline src/app/components/Footer/styled.tsx:97]: This one makes it harder to read to me, maybe use interpolation instead?

File: src/app/components/Footer/styled.tsx [hunk 2]
```diff
@@ -95,13 +103,21 @@ export const FooterWrapper = styled.footer`
...
-        0rem, calc(${toolbarWidth}rem - (100vw - ${contentWrapperMaxWidth}rem) / 2), ${toolbarWidth}rem
-      );
-    }
-  ` : ``}
+  ${props =>
+    props.isVerticalNavOpen === false
+      ? css`
+          @media (min-width: ${theme.breakpoints
+              .desktopBreak}em) and ${contentWrapperAndNavWidthBreakpoint} {
+            padding-left: clamp(
```
Comment (jivey) [inline src/app/components/Footer/styled.tsx:111]: The indentation here doesn't seem right, and to me, the original first line of the ternary reads much better without the extra line breaks and indentation.

File: src/app/components/Footer/index.tsx [hunk 3]
```diff
@@ -16,71 +16,145 @@ export const supportCenterLink = 'https://help.openstax.org/s/';
...
 
 // tslint:disable-next-line:variable-name
-const SocialIconMessage: React.FunctionComponent<{id: string, href: string, Icon: React.ComponentType}> =
-  ({id, href, Icon}) => <Styled.SocialIcon aria-label={useIntl().formatMessage({id})} href={href}
-    target='_blank' rel='noopener'>
+const SocialIconMessage: React.FunctionComponent<{
+  id: string;
+  href: string;
+  Icon: React.ComponentType;
+}> = ({ id, href, Icon }) => (
```
Comment (jivey) [inline src/app/components/Footer/index.tsx:64]: I know they are doing slightly different things, but the newline rules for this and https://github.com/openstax/rex-web/commit/1f1c8efa0557e37c46c7e08e47d19bd6b924bde9?w=0#diff-43286799a0a570580be50717cd6ba2a7a59d87e249a707fad745e4222daeaa26R167 seem inconsistent

**Review by jivey — CHANGES_REQUESTED** [review]
The icon changes look good, I'm not sure about the prettier changes though. I don't see a prettier file in root, but maybe we can adjust some settings. I left a few comments about some style things (that are subjective for sure.) I'm also wondering if we want to discuss a standard style as a team.

I use the built in formatter function that my LSP provides, and it mostly does what I want. That might be something the investigate for this, because as far as I understand, LSP formatters are more powerful?

File: src/app/components/Footer/styled.tsx (hunk 2 above)
Comment (RoyEJohnson) [reply src/app/components/Footer/styled.tsx:111]: This is sort of a nightmare of nesting.
There are some things I disagree with prettier about, but at least it is consistent, and IMO it does more good than harm. We get a lot of dense code.

**Comment by RoyEJohnson** [comment]
I was clear whether you were requesting that the prettier changes all be backed out, or just that some issues be addressed. I addressed the first two issues you highlighted.

**Review by jivey — APPROVED** [review]

**Pull request merged**